test_scripts/Defects/5_0/2596_Some_vehicle_data_params_are_Disallowed_after_Master_Reset_even_after_policies_update.lua

```diff
@@ -0,0 +1,88 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/SmartDeviceLink/sdl_core/issues/2596
+--
+-- Description:
+-- Some vehicle data params are Disallowed after Master Reset even after policies update (1586634)
+--
+-- Expected result: 
+-- SDL Defect:AppLink; Some vehicle data params are Disallowed after Master Reset even after 
+-- policies update
+---------------------------------------------------------------------------------------------------
```

Comment:
@KrupenichOleg Please update description in one of two ways:
1. In case .... SDL does ...
2. Steps ... Expected ...

Reply:
@dboltovskyi Done (commit [5331f8f](https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2123/commits/5331f8f3c612c9b4d6788bc19f4acf41b2279c75).)